Makefile

```
@@ -137,18 +140,18 @@ test-example:
 verify-image: ## Verifies operator image references and pull policy.
 .PHONY: verify-image
 verify-image:
-	./hack/verify-image.sh $(NEW_VERSION)
+	./hack/verify-image.sh "$(NEW_VERSION)"
 
 reset-image: ## Resets operator image references and pull policy.
 .PHONY: reset-image
 reset-image:
-	./hack/reset-image.sh $(IMAGE) $(OLD_VERSION)
+	./hack/reset-image.sh "$(IMAGE) $(OLD_VERSION)"
```

```suggestion
	./hack/reset-image.sh "$(IMAGE)" "$(OLD_VERSION)"
```
per https://github.com/openshift/contour-operator/blob/909d8a266eecb26d8cb06bddda6aefe20681212c/hack/reset-image.sh#L9-L10